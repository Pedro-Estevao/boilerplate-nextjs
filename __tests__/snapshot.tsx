import { render } from "@testing-library/react";
import Home from "@/app/page";
import { expect } from "@jest/globals";

it('renders homepage unchanged', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
});