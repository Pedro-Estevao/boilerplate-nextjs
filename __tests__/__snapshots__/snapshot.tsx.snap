// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    <main
      class="flex min-h-screen flex-col items-center justify-between p-24 bg-white dark:bg-dark"
    >
      <h1
        class="text-black dark:text-white"
      >
        Welcome to 
        <a
          href="https://nextjs.org"
        >
          Next.js!
        </a>
      </h1>
      <div
        class="sx-header-themeToggle ms-1 header-item d-none d-sm-flex"
      >
        <button
          class="btn btn-secondary btn-icon btn-topbar rounded-circle light-dark-mode"
          type="button"
        >
          ðŸŒ™
        </button>
      </div>
      <p
        class="text-black dark:text-white"
      >
        Apenas um teste
      </p>
    </main>
  </div>
</div>
`;
